### DISCUSSION TITLE
Is it even possible to write tests for Hyprland ?

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [hyprwm/Hyprland#4231] [General]
I just wanted to open a discussion on testing `Hyprland`, if it is possible and what would need to be done?

Small regressions and bugs might slip by when developing and changes could be made faster and with more confidence.

But there are SO many functionalities in `Hyprland` and SO many features that i find it difficult to think how would even test them all.

**Comment by vaxerski** [comment]
maybe, tho I cba
